README.md

```diff
@@ -8,21 +8,31 @@ The various configuration frameworks are intialised by supplying appsetting.json
 # Example Usage
 
 ```
-[Binding]
+[   [Binding]
     public class ContactUsConfigurationSetup
     {
         private readonly ScenarioContext _context;
         private readonly ObjectContext _objectContext;
         //private readonly IConfigurationRoot _configurationRoot;
+        //private readonly IConfigSection _configSection;
+        private readonly Configurator _configurator = new Configurator();
```

Comment:
I feel like this configurator class should be passed into this class through the constructor

Reply:
update pushed